pkg/channel/consolidated/reconciler/dispatcher/kafkachannel.go

```diff
@@ -148,49 +155,42 @@ func filterWithAnnotation(namespaced bool) func(obj interface{}) bool {
 }
 
 func (r *Reconciler) ReconcileKind(ctx context.Context, kc *v1beta1.KafkaChannel) pkgreconciler.Event {
-	logging.FromContext(ctx).Debugw("ReconcileKind for channel", zap.String("channel", kc.Name))
-	return r.syncDispatcher(ctx)
+	logging.FromContext(ctx).Debugw("ObserveKind for channel", zap.String("channel", kc.Name))
```

Comment:
`ObserveKind` ?  it's in the `ReconcileKind`:
https://github.com/knative-sandbox/eventing-kafka/blob/a23efea868056e51a111b4f79f265c769b136fae/pkg/channel/consolidated/reconciler/dispatcher/kafkachannel.go#L157-L160